ui/panels-plugin/src/plugins/time-series-chart/utils/data-transform.ts

```diff
@@ -152,24 +153,30 @@ export function convertPanelYAxis(inputAxis: YAxisOptions = {}): YAXisComponentO
     max: inputAxis?.max,
   };
 
+  // Set the y-axis minimum relative to the data
   if (inputAxis?.min === undefined) {
-    // Sets minimum axis label relative to data instead of zero.
+    // https://echarts.apache.org/en/option.html#yAxis.min
     yAxis.min = (value) => {
-      // https://echarts.apache.org/en/option.html#yAxis.min
       if (value.min >= 0 && value.min <= 1) {
         // Helps with PercentDecimal units, or datasets that return 0 or 1 booleans
         return 0;
       }
 
+      // Note: We can tweak the MULTIPLIER constants if we want
+      // TODO: Experiment with using a padding that is based on the difference between max value and min value
       if (value.min > 0) {
-        // Allows for padding between origin and first series.
-        // Current value was chosen arbitrarily and will need to be adjusted.
-        return value.min * MIN_VALUE_PADDING_MULTIPLIER;
+        return roundDown(value.min * POSITIVE_MIN_VALUE_MULTIPLIER);
+      } else {
+        return roundDown(value.min * NEGATIVE_MIN_VALUE_MULTIPLIER);
       }
-
-      // No padding added since negative numbers for min throws it off.
-      return value.min;
     };
   }
+
   return yAxis;
 }
+
+export function roundDown(num: number) {
```

Comment:
A quick comment with the function could be good so VSCode gives the nice summary on hover, for my own understanding, is this accurate for what it's doing?
```
/**
 * Rounds the input number down to the nearest power of 10.
 * Used to ensure the origin is always a whole number.
 */
```

Reply:
I like ittt